Error in running `make test`

I noticed that when I choose Python 3.5 environment, `pytest` complains about lack of `-n` argument and when I choose Python 3.6 environment and change the Makefile to allow it, I get the following error:

```
pytest -n 8 -x -m "not slow and not convert_maxpool"
Traceback (most recent call last):
  File "/usr/local/anaconda3/envs/venv_3.6/bin/pytest", line 8, in <module>
    sys.exit(main())
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/config/__init__.py", line 61, in main
    config = _prepareconfig(args, plugins)
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/config/__init__.py", line 182, in _prepareconfig
    config = get_config()
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/config/__init__.py", line 156, in get_config
    pluginmanager.import_plugin(spec)
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/config/__init__.py", line 530, in import_plugin
    __import__(importspec)
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/tmpdir.py", line 25, in <module>
    class TempPathFactory(object):
  File "/usr/local/anaconda3/envs/venv_3.6/lib/python3.6/site-packages/_pytest/tmpdir.py", line 35, in TempPathFactory
    lambda p: Path(os.path.abspath(six.text_type(p)))
TypeError: attrib() got an unexpected keyword argument 'convert'
make: *** [test] Error 1
```

At the moment I am using Python 3.5 environment and have changed the Makefile and removed the `-n 8`.

I am using conda for managing Pythons on Mac.

--- a/requirements.txt
+++ b/requirements.txt
@@ -13,7 +13,7 @@ termcolor==1.1.0
 sphinx-rtd-theme==0.4.2
 recommonmark==0.4.0
 google-cloud-storage==1.13.1
-pytest==4.2.0
+pytest==5.3.5
 pytest-xdist==1.26.1
 nose==1.3.7
 pyyaml==5.1

